ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-igmp.j2

```diff
@@ -10,10 +10,20 @@
 
 #### Router IGMP Summary
 
-| Settings | Value |
-| -------- | ----- |
+| VRF | SSM Aware | Host Proxy |
+| --- | --------- | ---------- |
 {%     if router_igmp.ssm_aware is arista.avd.defined %}
-| SSM Aware | {{ router_igmp.ssm_aware }} |
+| --- | Enabled | --- |
+{%     endif %}
+{%     if router_igmp.host_proxy_match_mroute is arista.avd.defined %}
+| default | --- | {{ router_igmp.host_proxy_match_mroute }} |
+{%     endif %}
+{%     if router_igmp.vrfs is arista.avd.defined %}
```

Comment:
This `if` can be skipped. It is already done inside the natural_sort filter, which will just return an empty list in case of `undefined or None`

Reply:
same in the config template

Reply:
removed the if loops